src/call/server.rs

```diff
@@ -117,14 +118,11 @@ impl RequestContext {
         self.ctx
     }
 
-    fn take_call(&mut self, cq: CompletionQueue) -> Option<Call> {
+    fn call(&mut self, cq: CompletionQueue) -> Call {
```

Comment:
Could `Call` lives longer than `RequestContext`? What will happen if call `grpc_call_unref` after `grpc_sys::grpcwrap_request_call_context_destroy`?

Reply:
> Could Call lives longer than RequestContext? 

Yes.

> What will happen if call grpc_call_unref after grpc_sys::grpcwrap_request_call_context_destroy?

Decreases ref counter and cancels the call if needed.